Cargo.toml

```diff
@@ -664,10 +665,11 @@ transforms-filter = []
 transforms-log_to_metric = []
 transforms-lua = ["dep:mlua", "vector-lib/lua"]
 transforms-metric_to_log = []
-transforms-pipelines = ["transforms-filter", "transforms-route"]
+transforms-pipelines = ["transforms-filter", "transforms-route", "transforms-exclusive-route"]
```

Comment:
```suggestion
```

Nit: I think that is a legacy feature that isn't used anymore (there used to be a `pipelines` transform that was removed).

Reply:
I will follow up with a PR to 🪓 this. Always happy to remove features.